PhpcsChanged/Cli.php

```diff
@@ -379,12 +398,6 @@ function runGitWorkflowForFile(string $gitFile, array $options, ShellOperator $s
 					$cache->setCacheForFile($gitFile, 'old', $oldFileHash, $phpcsStandard ?? '', $oldFilePhpcsOutput);
 				}
 			}
-		} else {
-			if ($isNewFile) {
-				$debug('Skipping the linting of the orig file version as it is a new file.');
-			} else {
-				$debug('Skipping the linting of the orig file version as the new version of the file contains no lint errors.');
```

Comment:
This logging is handled by the above `throw` and the below `catch`.

Reply:
Very elegant solution, which allowed a code serving the mere purpose of debugging messaging to be removed. Nice!